etcdctl/command/util.go

```diff
@@ -65,7 +65,7 @@ func getPeersFlagValue(c *cli.Context) []string {
 
 	// If we still don't have peers, use a default
 	if peerstr == "" {
-		peerstr = "127.0.0.1:4001"
+		peerstr = "127.0.0.1:4001,127.0.0.1:2379"
```

Comment:
I tried this. And it works just fine. The reason behind this is the default addr is a loopback, so it would never block long even if it is not listened.